packages/node_modules/sample-browser-auth-implicit/test/wdio/lib/set-value.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env browser */
+/**
+ * JWTs as access tokens are long enough that they have a significant impact on
+ * test duration when using browser.setValue. Instead, this function let's us
+ * inject them into the browser context and assign them in one shot.
+ *
+ * @param {string} selector
+ * @param {string} token
+ * @returns {undefined}
+ */
+export default function setValue(selector, token) {
```

Comment:
We're not injecting access tokens (which are stupid-long), so let's use wdio native setValue instead of this guy.